src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/ActionSvcClientImplTest.java

```diff
@@ -124,4 +129,106 @@ public void testCreateActionPlanRestClientException() {
+            .scheme(HTTP)
+            .host(LOCALHOST)
+            .port(80)
+            .path(ACTION_PATH)
+            .build();
+    when(restUtility.createUriComponents(any(String.class), any(MultiValueMap.class)))
+        .thenReturn(uriComponents);
+
+    ActionRulePostRequestDTO actionRulePostRequestDTO = new ActionRulePostRequestDTO();
+    actionRulePostRequestDTO.setName("BSREM+45");
+    actionRulePostRequestDTO.setActionTypeName("BSREM");
+    actionRulePostRequestDTO.setDescription("Enrolment Reminder Letter(+45 days)");
+    actionRulePostRequestDTO.setActionPlanId(
+        UUID.fromString("003e587a-843f-11e8-adc0-fa7ae01bbebc"));
+    actionRulePostRequestDTO.setTriggerDateTime(OffsetDateTime.now());
+    actionRulePostRequestDTO.setPriority(3);
+
+    ActionRuleDTO actionRuleDTO = new ActionRuleDTO();
+    actionRuleDTO.setName("BSREM+45");
+    actionRuleDTO.setActionTypeName("BSREM");
+    actionRuleDTO.setDescription("Enrolment Reminder Letter(+45 days)");
+    actionRuleDTO.setId(UUID.fromString("714356ba-7236-4179-8007-f09190eed323"));
+    actionRuleDTO.setTriggerDateTime(OffsetDateTime.now());
+    actionRuleDTO.setPriority(3);
+
+    HttpEntity httpEntity = new HttpEntity<>(actionRulePostRequestDTO, null);
+    when(restUtility.createHttpEntity(any(ActionRulePostRequestDTO.class))).thenReturn(httpEntity);
+    when(restTemplate.postForObject(
+            eq(uriComponents.toUri()), eq(httpEntity), eq(ActionRuleDTO.class)))
+        .thenReturn(actionRuleDTO);
+
+    // When
+    ActionRuleDTO createdActionRuleDTO =
+        actionSvcClient.createActionRule(
+            actionRulePostRequestDTO.getName(),
+            actionRulePostRequestDTO.getDescription(),
+            actionRulePostRequestDTO.getActionTypeName(),
+            actionRulePostRequestDTO.getTriggerDateTime(),
+            actionRulePostRequestDTO.getPriority(),
+            actionRulePostRequestDTO.getActionPlanId());
+
+    // Then
+    verify(restTemplate)
+        .postForObject(eq(uriComponents.toUri()), eq(httpEntity), eq(ActionRuleDTO.class));
+    verify(restUtility).createHttpEntity(eq(actionRulePostRequestDTO));
+    assertEquals(createdActionRuleDTO.getName(), actionRuleDTO.getName());
+    assertEquals(createdActionRuleDTO.getDescription(), actionRuleDTO.getDescription());
+    assertEquals(createdActionRuleDTO.getActionTypeName(), actionRuleDTO.getActionTypeName());
+    assertEquals(createdActionRuleDTO.getId(), actionRuleDTO.getId());
+  }
+
+  /**
+   * Test that a rest client exception is thrown when issues contacting the action service to create
+   * an action rule.
+   */
+  @Test(expected = RestClientException.class)
+  public void testCreateActionRuleRestClientException() {
+    // Given
+    ActionSvc actionSvcConfig = new ActionSvc();
+    actionSvcConfig.setActionRulesPath(ACTION_RULE_PATH);
+    when(appConfig.getActionSvc()).thenReturn(actionSvcConfig);
+
+    UriComponents uriComponents =
```

Comment:
Does defining the UriComponents add any value to this test? I think this could just be 

```
    when(restTemplate.postForObject(
            eq(any(), any(), any())
        .thenThrow(RestClientException.class);

     // When
    actionSvcClient.createActionRule(
        "BSREM+45",
        "Enrolment Reminder Letter(+45 days)",
        "BSREM",
        OffsetDateTime.now().plusDays(45),
        3,
        UUID.fromString("003e587a-843f-11e8-adc0-fa7ae01bbebc"));
     // Then RestClientException is thrown
```